resources/lib/http.py

```diff
@@ -5,10 +5,12 @@
 
 from .stream import Streams
 from .recording import Recordings
+from .relive import Relives, ReliveRecordings
 from . import gui
 
 BASE_URL = 'https://media.ccc.de/public/'
 LIVE_URL = 'https://streaming.media.ccc.de/streams/v2.json'
+RELIVE_URL = 'http://relive.c3voc.de/relive/index.json'
```

Comment:
I think we should use https://cdn.c3voc.de/relive/index.json, that https URL redirects to the correct relive URL afterwards.